.github/pull_request_template.md

```diff
@@ -0,0 +1,22 @@
+<!--
+Add the ticket number below and uncomment
+[Link to Jira ticket](https://beisdigital.atlassian.net/browse/PC-####)
+-->
+
+# Description
+
+ <!-- Add a brief description of the change(s) you have made --> 
+
+# Checklist
+
+- [ ] I have made any necessary updates to the documentation
+- [ ] I have checked there are no unnecessary IDE warnings in this PR
+- [ ] I have checked there are no unintentional line ending changes
+- [ ] I have added tests where applicable
+- [ ] If I have made any changes to python files, I have run `make extract-translations`. 
```

Comment:
Is this all Python files, or can we be more specific?

Reply:
essentially. in theory a more correct description is 'all Python files that contain a string to be translated, even if the string wasn't edited' but in practice this is most python files so I left as all, no harm as regenerating translations doesn't lead to any changes

Reply:
Fair enough. In that case maybe it's worth emphasising the point:
```
- [ ] If I have made any changes to Python files (even if not changing any content strings), I have run `make extract-translations`.
```
?

Reply:
sure